packages/dotcom-ui-header/src/__stories__/story.tsx

```diff
@@ -55,6 +55,31 @@ DefaultHeaderWithDrawer.args = {
   showLogoLink: false
 }
 
+export const EnhancedHeaderWithDrawer = (args) => (
+  <OnReady callback={onReadyCallback}>
+    <HeaderSimple {...storyData} {...args} />
+    <Drawer {...storyData} {...args} />
+    <p className="demo-sticky-message demo-sticky-message--scroll">Scroll down</p>
+  </OnReady>
+)
+
+EnhancedHeaderWithDrawer.story = {
+  name: 'Enhanced header with drawer'
+}
+EnhancedHeaderWithDrawer.args = {
+  showSubNavigation: true,
+  showMegaNav: true,
+  showUserNavigation: true,
+  userIsLoggedIn: false,
+  userIsSubscribed: false,
+  showLogoLink: false,
+  enhancedSearchProps: {
+    suggestionsApiUrl: 'https://www.ft.com/search-api/suggestions?count=5&partial=',
+    searchUrl: 'https://enhanced-search-staging.ft.com/?q=',
```

Comment:
Why we use the staging link? Is not it better to use the production one (`https://enhanced-search.ft.com/?q=` ?

Reply:
This is just for the story, and can be changed in the storybook controls, or we can even remove the story all together